tests/php/ORM/DataListTest.php

```diff
@@ -753,7 +753,7 @@ public function testSimpleFilterStartsWith()
 
     /**
      * @expectedException \SilverStripe\Core\Injector\InjectorNotFoundException
-     * @expectedExceptionMessage Class DataListFilter.Bogus does not exist
+     * @expectedExceptionMessageRegExp /Class "?DataListFilter.Bogus"? does not exist/
```

Comment:
In the newer PHP units, these annotations are deprecated and removed - it may be worth moving them to method calls.

Reply:
Good call. If you do that, you could potentially have a separate string used for PHP 8, which would clarify when the different strings are used, but make the test more verbose. Any views on which are better?

Reply:
I'd be happy to keep the regex and slightly less verbose rather than a bunch of `if PHP_VERSION > 8` conditions.

Reply:
Agreed, I've gone with retaining the regex. Syntax switched to method calls.